tests/foreman/cli/test_contentview.py

```diff
@@ -682,6 +713,45 @@ def test_positive_create_composite_with_component_ids(self):
             'the input values',
         )
 
+    @tier2
+    @run_only_on('sat')
+    def test_positive_update_composite_with_component_ids(self):
+        """Update a composite content view with a component_ids option
+
+        @id: e6106ff6-c526-40f2-bdc0-ae291f7b267e
+
+        @assert: Composite content view component ids are similar to the
+        nested content view versions ids
+
+        @CaseLevel: Integration
+        """
+        # Create a CV to add to the composite one
+        inserted_cv = make_content_view({u'organization-id': self.org['id']})
```

Comment:
`can we rename `inserted_cv` to some other name ? may be leave it as  `cv` or `cv1` ?